.travis.yml

```diff
@@ -40,6 +53,14 @@ deploy:
       branch: master
       tags: false # Disable publishing for release tags
 
+  # Publishing release artifacts
+  - provider: script
+    skip_cleanup: true
+    script: scripts/publish-release
```

Comment:
`sbt +publish` ?

Reply:
The scripts are printing some more context. I think it preferable to keep it.